extensions/wikia/Wall/maintenance/CleanupCommentsIndex.php

```diff
@@ -20,11 +27,13 @@ public function execute() {
 			}, $ids);
 
 			\Wikia\Logger\WikiaLogger::instance()->info(
-				"CleanupCommentsIndex: " . count( $ids ) . " rows affected",
+				"{$wgDBname}: " . count( $ids ) . " rows affected",
 				[ "rows_count" => count( $ids ) ]
 			);
 
-			$db->delete( 'comments_index',  "comment_id in (" . implode( ',', $ids ) . ")");
+			if ( !$this->hasOption('dry-run') ) {
+				$db->delete( 'comments_index', "comment_id in (" . implode( ',', $ids ) . ")" );
```

Comment:
You can provide delete query conditions as an assoc array:

```php
[
  'comment_id' => $ids
]
```

MediaWiki database handling layer will expand it to `foo IN (...) condition. Just please check whether `$ids` has at least one item - otherwise MW throws an exception.